python/sparkdl/transformers/utils.py

```diff
@@ -36,3 +34,147 @@ class ImageNetConstants:
 class InceptionV3Constants:
     INPUT_SHAPE = (299, 299)
     NUM_OUTPUT_FEATURES = 131072
+
+
+class TFInputGraph(object):
+    """
+    An opaque serializable object containing TensorFlow graph.
+    """
+    # TODO: for (de-)serialization, the class should correspond to a ProtocolBuffer definition.
+    def __init__(self, graph_def, input_mapping, output_mapping):
```

Comment:
Talked offline -- TFInputGraph should be a super thin wrapper containing graph_def. It's constructor really shouldn't be called by the user and they should use TFInputGraphBuilder below -- we should at least document this and warn that the API could change for the constructor here.